src/Http/Http/src/StreamPipeReader.cs

```diff
@@ -8,7 +8,6 @@
 using System.IO.Pipelines;
 using System.Runtime.CompilerServices;
 using System.Runtime.ExceptionServices;
-using System.Runtime.InteropServices;
```

Comment:
I would leave out these changes since this type is going to be moving to corefx.

Reply:
:ok: I'll revert changes to `StreamPipeReader` and `StreamPipeWriter`.

Will CoreFx also take `StreamPipeReaderTests` and `StreamPipeWriterTests`? If yes, I"ll revert changes to those classes too.

Reply:
Yes they will.